api.web.yaml

```
@@ -18,6 +18,15 @@ paths:
                 $ref: "#/components/schemas/ProfileResponse"
         default:
           description: Unexpected error
+  /profile/agreements:
+    post:
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/AgreementConformation"
```

```suggestion
              $ref: "#/components/schemas/AgreementConfirmation"
```

Or was `conformation` deliberate here?

---

🤦🏼 – thanks